Simple.java

```diff
@@ -0,0 +1,43 @@
+import java.io.*;
+import java.util.*;
+
+class Simple {
+    static HashMap<String, Integer> countMap = new HashMap<>();
+
+    public static void main(String[] args) throws IOException {
+        String str;
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

```

Comment:
Nits:

* What about calling `str` -> `line` instead?
* Can we make these `var input = ...` and `var output =` respectively? Seems easier to read than `br` and `bw`.